README.md

```diff
@@ -44,22 +44,56 @@ This repository has the following structure:
+    │   │        │       └── thumbnail.png                      # A representation of the rendered version of your template (recommended size: 164 x 217)
+    │   │        └── ...
+    │   └── repositories                                        # The folder with a repository definition.
+    │        ├── {repositories-type}                            #  Contains a repository that has a cloud storage (AWS S3 or Azure) as the underlying storage
+    │        │    ├── {repository-name}
+    │        │    │   ├── format.txt                            # Applicable package format
+    │        │    │   ├── package.txt                           # List of items to be imported into a repository
+    │        │    │   └── repository.xml                        # A repository definitions
+    │        │    └── ...
+    │        └── ...
+    └── business                                                # Business collection
+        ├── general-notifications                               # General notifications related collection
+        │   ├── event-general-notifications     
+        │   │   ├── {event-general-notifications-name}              
+        │   │   │   ├── {general-notification-name}             # The folder to store channel specific data for general notifications   
+        │   │   │   │   ├── in-app-notification                         
+        │   │   │   │   │   ├── {locale}                        # The folder to store locale-specific engagement templates
+        │   │   │   │   │   │   └── engagement-template.json    # The engagement template that contains default messages with the Handlebars expressions
+        │   │   │   │   │   └── in-app_channel-settings.json    # The channel settings for in-app notification channels per locale
+        │   │   │   │   └── push
+        │   │   │   │       ├── {locale} 
+        │   │   │   │       │   └── engagement-template.json 
+        │   │   │   │       └── in-app_channel-settings.json     
+        │   │   │   └── event-general-notifications.json        # The event with one or more general-notification definitions
+        │   │   └── ... 
+        │   ├── templates                                       # The folder to store templates
+        │   │   ├── {template-name}
+        │   │   │   ├── content-schema.json                     # A structure of an engagement template based on a given template
+        │   │   │   └── template.json                           # A template definition
+        │   │   └── ...
+        │   └── placeholders                                    # The folder to store placeholder group
+        │       ├── {placeholder-group-name}
+        │       │   └── placeholders.json                       # A placeholder group configuration
+        │       └── ...
+        └── repositories                                        # The folder with a repository definition.
```

Comment:
Please check again the repositories configuration folder and the create-collections.sh file with what we have on ModelBank